soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -11171,7 +11171,20 @@ void Player_DrawGameplay(PlayState* play, Player* this, s32 lod, Gfx* cullDList,
             MATRIX_TOMTX(sp70);
         }
 
-        gSPDisplayList(POLY_OPA_DISP++, sMaskDlists[this->currentMask - 1]);
+       
+        if ((this->currentMask == PLAYER_MASK_BUNNY) && CVarGetInteger("gCosmetics.Equipment_BunnyHood.Changed", 0)) {
+            gSPGrayscale(POLY_OPA_DISP++, true);
```

Comment:
Why not put the grayscale commands into the DList patching so it becomes all self contained? Bria added the ability to "add" extra instructions to a gfx patch with the fix for greg bridge, so that could probably be used.

Reply:
@Patrick12115 https://github.com/Patrick12115/Shipwright/pull/5